doc/user/content/releases/v0.32.md

```diff
@@ -49,3 +46,20 @@ This version of Materialize is not yet released.
   [AWS PrivateLink connections](/sql/create-connection/#aws-privatelink),
   which establish links to
   [AWS PrivateLink](https://aws.amazon.com/privatelink/) services.
+
+## Patch releases
+
+### v0.32.1
+
+* Stabilize the performance of ad hoc `SELECT` statements against unindexed
+  objects in large clusters {{% gh 16090 %}}.
+
+### v0.32.2
+
+* Fix a memory leak caused by lingering read handles when dropping materialized
+  views or indexes {{% gh 16127 %}}.
+
+### v0.32.3
+
+* Fix a bug in predicate pushdown that could result in incorrect query plans {{% gh
```

Comment:
This actually made it into v0.32.2. I just messed up the backport and the tests were failing for an unimportant reason, so I had to recut as v0.32.3. Not sure what the right answer is here. We didn't roll out v0.32.1 or v0.32.2 to anyone, so maybe we just document v0.32.3?

Reply:
I noticed that v0.32.3 only included the change in the test, but it was...way too complicated to break it down into patches then. :troll-face-peeking: I agree that it'd make more sense to bulk the fixes that go into patch releases, but also don't have a good suggestion for the least bad way of doing it. Need to sleep on it?

Reply:
Rolled it all under v0.32.3.